core-errors/src/lib.rs

```diff
@@ -116,7 +110,10 @@ pub enum TerminationReason {
 }
 
 #[derive(Debug, Clone, Copy, Eq, PartialEq, Hash, derive_more::Display)]
+#[cfg_attr(feature = "codec", derive(Encode, Decode, TypeInfo))]
 pub enum ExtError {
+    #[display(fmt = "Unknown error")]
+    Unknown,
```

Comment:
Maybe `SomeError`? I mean we need such a name, that emphasize that there is actually an error, but you don't know exactly the variant